libkbfs/kbfs_ops_test.go

```diff
@@ -56,6 +56,7 @@ var individualTestTimeout = 10 * time.Second
 
 func kbfsOpsInit(t *testing.T, changeMd bool) (mockCtrl *gomock.Controller,
 	config *ConfigMock, ctx context.Context, cancel context.CancelFunc) {
+	t.Log("Init KBFSOps")
```

Comment:
Do you think we need to keep this in, or is it just leftover personal debugging?  Seems like it wouldn't be too useful most of the time, but I don't care too much if you want to keep it.